ingestion-edge/ingestion_edge/publish.py

```diff
@@ -3,44 +3,135 @@
 # file, you can obtain one at http://mozilla.org/MPL/2.0/.
 
 from .conf import ROUTE_TABLE
+from .conf import QUEUE_PATH
 from datetime import datetime
 from flask import Blueprint, request
-from google.cloud.pubsub import PublisherClient
+from google.api_core.exceptions import Aborted
+from google.api_core.exceptions import Cancelled
+from google.api_core.exceptions import Forbidden
+from google.api_core.exceptions import RetryError
+from google.api_core.exceptions import ServerError
+from google.api_core.exceptions import TooManyRequests
```

Comment:
Since we only use each of these once to build the `TRANSIENT_ERRORS` tuple, I'd recommend importing the package and using fully qualified names:

```
import google.api_core.exceptions

TRANSIENT_ERRORS = (
    # Comment
    google.api_core.exceptions.Forbidden,
    # Comment
    google.api_core.exceptions.RetryError,
...
```